src/test/java/picocli/AutoCompleteTest.java

```diff
@@ -16,15 +16,9 @@
 package picocli;
 
 import org.hamcrest.CoreMatchers;
-import org.junit.Ignore;
 import org.junit.Rule;
 import org.junit.Test;
-import org.junit.contrib.java.lang.system.Assertion;
-import org.junit.contrib.java.lang.system.ExpectedSystemExit;
-import org.junit.contrib.java.lang.system.ProvideSystemProperty;
-import org.junit.contrib.java.lang.system.RestoreSystemProperties;
-import org.junit.contrib.java.lang.system.SystemErrRule;
-import org.junit.contrib.java.lang.system.SystemOutRule;
+import org.junit.contrib.java.lang.system.*;
```

Comment:
I prefer not to do star * imports. Can you keep the existing explicit imports?

Reply:
No problem. It seems that I run the "Optimize Imports" shortcut by mistake.